### DISCUSSION TITLE
Is Chakra v3 sidelined?

### DISCUSSION THREAD

**Discussion by MikeWillis** [chakra-ui/chakra-ui#8381] [Q&A] [answered]
Hi all,

I'm going to be starting a new project soon that I would normally use chakra for. Trying to decide if I should use chakra 2, or maybe panda + ark, or wait a bit for chakra 3. Anyone know the latest on 3?

**Comment by segunadebayo** [comment, marked as answer]
We’re aiming for next weeks to complete v3 tasks. Feel free to use v2 for now, we’ll make sure the migration guide is as comprehensive as possible.

**Reply by MikeWillis** [reply]
great to hear, thank you @segunadebayo and everyone else over there! chakra is awesome thanks to you all!

**Reply by vitalijalbu** [reply]
@segunadebayo hi, any new components coming?

**Reply by segunadebayo** [reply]
@vitalijalbu Please don’t ask this question over and over 😆

As I’ve always said. We aim to include all components in Ark UI into v3

**Reply by DavidCodesDev** [reply]
@segunadebayo Is the plan still to release a v4 shortly after? Cause if you include all ARK components to V3 then V4 is not needed anymore right? (cause that was the plan iirc)

Btw I love how your name looks like Segunda (a form in Bleach) and ''bayo'' sounding like DATEBAYO from naruto 😄
